data-prepper-plugins/elasticsearch/README.md

```diff
@@ -8,7 +8,7 @@ The Elasticsearch sink should be configured as part of Data Prepper pipeline yam
 
 ### <a name="raw_span_trace_analytics"></a>Raw span trace analytics
 
-```$xslt
+```
```

Comment:
Oh, you can specify the language of the code block (in this case YAML) by doing:
* \```yaml

So just replace "$xslt" with "yaml" here and elsewhere.